lib/node-utils/is-node-of-type.ts

```diff
@@ -49,3 +49,4 @@ export const isReturnStatement = isNodeOfType(AST_NODE_TYPES.ReturnStatement);
 export const isFunctionExpression = isNodeOfType(
   AST_NODE_TYPES.FunctionExpression
 );
+export const isAwaitExpression = isNodeOfType(AST_NODE_TYPES.AwaitExpression);
```

Comment:
Same as the other PR: this is available in `@typescript-eslint/experimental-utils`

Reply:
Nice :)